contributors/design-proposals/thirdpartyresources.md

```diff
@@ -0,0 +1,224 @@
+# Moving ThirdPartyResources to beta
+
+## Background
+There are a number of important issues with the alpha version of
+ThirdPartyResources that we wish to address to move TPR to beta. The list is
+tracked [here](https://github.com/kubernetes/features/issues/95), and also
+includes feedback from existing Kubernetes ThirdPartyResource users. This
+proposal covers the steps we believe are necessary to move TPR to beta and to
+prevent future challenges in upgrading.
+
+
+## Goals
+1. Ensure ThirdPartyResource authors can version their APIs and offer their
+users a safe migration path between versions (including alpha->beta for the
```

Comment:
@smarterclayton I explicitly punt on this in the next sub-section.

Maybe something more like `Ensure ThirdPartyResource APIs operate consistently with first party Kubernetes APIs`?